tsconfig.json

```diff
@@ -15,7 +15,6 @@
     "skipLibCheck": true
   },
   "files": [
-    "src/index.ts",
-    "tests/index.ts"
+    "src/index.ts"
```

Comment:
not including the test files in the compilation may be why the tests were failing. I'll have to check locally first